types/office-js-preview/index.d.ts

```diff
@@ -521,6 +525,15 @@ declare namespace Office {
         */
         value: T;
     }
+    /**
+     * Used to associate an action.
+     */
+    interface Actions {
+        /**
+         * Function to associate a action name with the action function.
+         */
+        associate: (actionName: string, action: (arg?: any) => void) => void;
```

Comment:
Can the `arg` type really be `any`? Could it be a Function? If not, then maybe the type should be `Object`. Can it be any object, or does it have to be an Office object of some type? 
If this type can be made stronger, then we should do that. The stronger the typing, the more useful the intellisense will be.

Reply:
Arg is only supported for the first-party case and It can be any object which doesn't have to be an office-specific object.